gensrc/thrift/Descriptors.thrift

```diff
@@ -139,7 +139,8 @@ enum TSchemaTableType {
     SCH_TABLE_PROPERTIES = 50,
     SCH_FILE_CACHE_STATISTICS = 51,
     SCH_CATALOG_META_CACHE_STATISTICS = 52;
-    SCH_ROUTINE_LOAD_JOB = 53;
+    // consistent with the master
+    SCH_ROUTINE_LOAD_JOB = 54;
```

Comment:
JOBS？

Reply:
After https://github.com/apache/doris/pull/49515 pick to branch-3.0